ApollosProject/apollos-prototype
# Adds Search indexing and Job Queue

## DESCRIPTION

### What does this PR do, or why is it needed?

- Adds a background job queue and a "full index" worker that runs every day at 1:00. 

### What design trade-offs/decisions were made?

Punting delta indexing to another PR.

### How do I test this PR?

1. Clone this PR, `yarn install`
2. Uncomment this line: https://github.com/ApollosProject/apollos-prototype/pull/1062/files#diff-0f14ef6d2428a705a3936d9335d597f4R11
3. Be ready to open the Job Dashboard http://localhost:4000/admin/queues
4. Make sure you have redis running locally. 
5. Start the server
6. You should see one scheduled job and one completed job in the dashboard.

### What automated tests did you write?

Tests over datasource.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
